guides/hack/05-functions/05-format-strings.md

```diff
@@ -1,6 +1,6 @@
-The Hack typechecker can also check that format strings are being used
-correctly.
+The Hack typechecker can check that format strings are being used correctly.
```

Comment:
This is already a wording improvement, but "can" implies it's optional. How about "The Hack typechecker will check that ..." or "The Hack typechecker checks that ..."?

Reply:
`The Hack typechecker checks that` is better. Updating.